docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -10,799 +10,766 @@
 title: LoopBack 4
 url: index.html
 children:
-
```

Comment:
Please revert the formatting changes and only leave your addition using the current format.